docs/director/03-error-handling.md

```diff
@@ -1,23 +1,39 @@
 # Error handling
 
-Director API returns custom errors when there are problems with handling the request. The errors can be divided into
-two categories: 
-- Regular HTTP errors 
-- GraphQL errors
-
-## Regular errors
-
-Regular errors are related to the request itself. They may concern issues such as missing headers. These are the regular errors that may occur when calling the Director API:
-
-- **Tenant not found**
-
-  This error occurs either when there is no such tenant in the database, or when the passed tenant does not match the tenant in the [System Auth](https://github.com/kyma-incubator/compass/blob/732486482e1d71384be4d705b1ed260365f74c1c/docs/compass/03-01-security.md#system_auths-table). 
-
+Director API returns custom errors when there are problems with handling the request. 
 
 ## GraphQL errors
 
-GraphQL errors are related to the request execution. They may concern issues with connecting to the database, or inter-server issues. These are the GraphQL errors that may occur when calling the Director API:
-
-- **Tenant is required**
-
-  This error occurs when a tenant, that is required by the operation, wasn't passed.
+These are the GraphQL errors that may occur when calling the Director API:
+
+| Error type             | Error code    |                            Description                                                                      |
+|------------------------|---------------|-------------------------------------------------------------------------------------------------------------|
+| `InternalError`        | `10`          | Specifies any error that cannot be handled in the Director. Such situation is unique and serious.           |
+| `NotFound`             | `20`          | Indicates that a given resource cannot be found and further processing is impossible.                       |
+| `NotUnique`            | `21`          | Indicates that a given resource is not unique.                                                              |
+| `InvalidData`          | `22`          | Indicates that the input data is invalid with the reason described in the error message.                    | 
+| `InsufficientScopes`   | `23`          | Indicates that the client doesn't have permission to execute the operation.                                 |
+| `TenantRequired`       | `24`          | Indicates that a tenant is not found in the request, which is required to successfully handle the request.  |
+| `TenantNotFound`       | `25`          | Indicates that the internal tenant is not found in the Director.                                            |
+| `Unauthorized`         | `26`          | Indicates that request cannot be authorized.                                                                |            
+| `InvalidOperation`     | `27`          | Indicates that the operation is invalid, because of restrictions, eg.: cannot delete label definition, where the label is used.   |
```

Comment:
```suggestion
| `InvalidOperation`     | `27`          | Indicates that the operation is invalid because of certain restrictions, e.g.: you cannot delete a given label definition when the label is used.   |
```
`where` or `when`? Is `cannot delete label definition, where the label is used` an example of a situation or an example of an error message?

Reply:
Is an example of situation.